api/filter_outputs.go

```diff
@@ -41,9 +41,49 @@ func (api *API) updateFilterOutput(w http.ResponseWriter, r *http.Request) {
 			ctx,
 			w,
 			statusCode(err),
-			errors.Wrap(err, "failed to create filter outputs"),
+			errors.Wrap(err, "failed to create filter output"),
 		)
 		return
 	}
 	api.respond.StatusCode(w, http.StatusOK)
 }
+
+//createFilterOutputEvent will add a new event to the list of existing events in the filter outputs
+func (api *API) addFilterOutputEvent(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "filter_output_id")
+
+	var req createEventRequest
```

Comment:
Please keep the contract models precise to the handler names. `addFilterOutputEvent` -> `addFilterOutputEventRequest` etc